src/sdsc/__init__.py

```diff
@@ -202,6 +190,7 @@ def termcheck(context, termfileid, content, contentpretty, contextid, basefile,
                 replacedtokens = findnumberoftokens.search(word)
                 if replacedtokens:
                     currenttokeninparagraph += int(replacedtokens.group(0)) - 1
+                continue
```

Comment:
This looks wrong to me -- in termcheck(), we do not want to skip `##@...##` stuff, there may be patterns that look for these (for examples, do a `grep -n "##@" xsl-checks/*.xml`).